researcher_ids/RI_Claims_V1.md

```diff
@@ -279,41 +393,162 @@ interfaces, such as:
+
+-   Validation, as outlined elsewhere in this specification and the
+    GA4GH AAI Specification, MUST be performed before Passport Visas are
+    used for identity or authorization.
 
--   For example, ga4gh.ResearcherStatus.value =
-    "<https://doi.org/10.1038/s41431-018-0219-y>” represents the Registered
-    Access Bona Fide researcher status.
+### Passport Visa Format
 
--   For the purposes of enforcing its policies for access, a Claim Clearinghouse
-    evaluating the "value" field MUST:
+Passport Visas are JWS Compact Serialization strings of the following
+form when represented in JSON:
+
+```
+{
+  "typ": "JWT",
+  "alg": "RS256",
+  ["jku": "<JKS-URL>",]
+  "kid": "<key-identifier>"
+}.
+{
+  "iss": "<issuer-URL>",
+  "sub": "<subject-identifier>",
+  ["scope": "openid ...",]
+  "jti": "<token-identifier>",
+  "iat": <seconds-since-epoch>,
+  "exp": <seconds-since-epoch>,
+  "ga4gh_visa_v1": {
+    "type": "<passport-visa-type>",
+    "asserted": <seconds-since-epoch>,
+    "value": "<value-string>",
+    "source": "<source-URL>",
+    ["condition": {...},]
+    ["by": "<by-identifier>"]
+  }
+}.<signature>
+```
+
+Where fields within the `ga4gh_visa_v1` object are as described in the
+[Passport Visa Fields](#passport-visa-fields) section of this
+specification except for "exp" which is a direct JWT claim and not
+stored within the `ga4gh_visa_v1` object.
+
+One of `scope` or `jku` MUST be present as described in
+[Conformance for Embedded Token Signatories](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#conformance-for-embedded-token-signatories)
+within the [AAI specification](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md).
+
+#### "**exp**"
+
+-   REQUIRED. Generally, it is seconds since unix epoch of when the
+    [Passport Assertion Source](#passport-assertion-source) requires such a
+    claim to be no longer valid. A Passport Assertion Source MAY choose
+    to make a claim very long lived. However, a [Passport Visa
+    Signatory](#passport-visa-signatory) MAY choose an earlier timestamp
+    in order to limit the claim’s duration within downstream Passport
+    Clearinghouses.
+
+-   Access is NOT necessarily removed by the `exp` timestamp. Instead,
+    this timestamp may be viewed as a cut-off after which no new access
+    will be granted and action to remove any existing access may
+    commence anytime shortly after this cut-off period.
+
+-   Its use by a Passport Clearinghouse is described in the
+    [Passport Visa Expiry](#passport-visa-expiry) section and
+    [Token Revocation](#token-revocation) section.
+
+## Passport Visa Fields
+
+Although standard fields within a [Passport Visa Object](#passport-visa-object)
+are defined in this section, other fields MAY exist within the object
+and should be ignored by any Passport Clearinghouse that is not familiar
+with the use of such fields. Field names are reserved for definition by
+the GA4GH DURI committee.
```

Comment:
Can we assume DURI exists forever (or at least as long as this spec?) Would it be more future-proof to refer to GA4GH only?

Reply:
Opened an issue to track this